### PR TITLE
[Forge] Fix local mode forge run

### PR DESCRIPTION
**Pull request by sitalkedia**
### Description

In local mode, submit lower TPS so that the transaction emitter is stable and also don't check for TPS/latency criteria.

### Test Plan

```
FORGE_RUNNER_MODE=local ./testsuite/run_forge.sh
```

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2153)

### CODE REVIEW

File: testsuite/run_forge.sh [hunk 1]
```diff
@@ -168,15 +169,19 @@ if [ -n "$AVG_TPS" ]; then
     echo "forge_job_avg_tps {FORGE_CLUSTER_NAME=\"$FORGE_CLUSTER_NAME\",FORGE_NAMESPACE=\"$FORGE_NAMESPACE\",GITHUB_RUN_ID=\"$GITHUB_RUN_ID\"} $AVG_TPS" | curl -u "$PUSH_GATEWAY_USER:$PUSH_GATEWAY_PASSWORD" --data-binary @- ${PUSH_GATEWAY}/metrics/job/forge
     if [[ "$AVG_TPS" -lt "$TPS_THRESHOLD" ]]; then
         echo "(\!) AVG_TPS: ${avg_tps} < ${TPS_THRESHOLD} tps"
-        FORGE_EXIT_CODE=1
+	if [ "$FORGE_RUNNER_MODE" != "local" ]; then
```
Comment (rustielin) [inline testsuite/run_forge.sh:172]: Can we move this perf threshold checking into the performance test itself ?

File: testsuite/run_forge.sh [hunk 2]
```diff
@@ -89,7 +89,8 @@ if [ "$FORGE_RUNNER_MODE" = "local" ]; then
     # more file descriptors for heavy txn generation
     ulimit -n 1048576
 
-    cargo run -p forge-cli -- test k8s-swarm \
+    cargo run -p forge-cli -- --workers-per-ac 10 \
+	 test k8s-swarm \
```
Comment (rustielin) [inline testsuite/run_forge.sh:93]: Nit indent

**Review by rustielin — CHANGES_REQUESTED** [review]

File: testsuite/run_forge.sh (hunk 1 above)
Comment (sitalkedia) [reply testsuite/run_forge.sh:172]: We can do that, but that will require some plumbing in the transaction emitter. May be as a follow up I can take a look at this?

Comment (rustielin) [reply testsuite/run_forge.sh:176]: hm ok yea. this script is just getting too complicated

**Review by rustielin — APPROVED** [review]

**Pull request merged**